src/components/molecules/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -78,17 +83,22 @@ export const HamburgerMenu = ({
 }: {
     links: { label: string; link: string }[];
 }) => {
-    const [showMore, setShowMore] = useState(false);
+    const [showMore, setShowMore] = useState<boolean>(false);
+    const [showTradingSublinks, setShowTradingSublinks] =
+        useState<boolean>(false);
 
     return (
         <Menu>
-            {({ close }) => (
+            {({ open, close }) => (
                 <>
                     <Menu.Button aria-label='Hamburger Menu'>
-                        <Burger className='h-6 w-6 tablet:h-8 tablet:w-8' />
```

Comment:
Removed tablet size. It's the same size as mobile: https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Responsive-Designs-(Post-Mainnet-Launch)?node-id=15838-37819&m=dev